jsonnet/rules.libsonnet

```
@@ -180,7 +180,7 @@ local droppedKsmLabels = 'endpoint, instance, job, pod, service';
                   cluster:master_nodes
                   * on(node) group_left() max by(node)
                   (
-                    kube_node_status_capacity_memory_bytes
+                    kube_node_status_capacity{unit="byte"}
```

same here but with memory

```suggestion
                    kube_node_status_capacity{resource="memory", unit="byte"}
```